themes/dark_spinel.json

```diff
@@ -342,7 +343,8 @@
         "storage.type.function.js",
         "storage.type.js",
         "storage.type.function.arrow.js",
-        "constant.language.null.js"
+        "constant.language.null.js",
+        "support.type.builtin.rbs"
```

Comment:
Or should it be in `Keywords`?

Reply:
I think it makes sense to put it together with the other type related tokens. The only distinction is the boldness anyway.